src/PJ_sch.c

```diff
@@ -0,0 +1,206 @@
+/*
+** Author : Piyush Agram
+** This code is in the public domain and implements the SCH coordinate system first developed at JPL.
```

Comment:
Public domain is not really a valid license. And it's going to be confusing if it's not under the same license as the rest of the code.